lib/bkblz/v1/request.rb

```diff
@@ -78,7 +77,13 @@ def fetch(http, request, limit=10)
         # You should choose a better exception.
         raise TooManyRedirectError, 'too many HTTP redirects' if limit == 0
 
-        response = http.start { |http| http.request(request) }
+        old_verbose = $VERBOSE
+        $VERBOSE = false
+        begin
+          response = http.start { |http| http.request(request) }
+        ensure
+          $VERBOSE = old_verbose
+        end
```

Comment:
i'm really not a fan of this change. is the only problem here that a warning is generated under $VERBOSE?

this doesn't actually fix the warning, but just hides it and any other problems during the request